assets/src/components/inactiveNotificationModal.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { useContext } from "react"
+import { NotificationsContext } from "../contexts/notificationsContext"
+import { StateDispatchContext } from "../contexts/stateDispatchContext"
+import { closeIcon } from "../helpers/icon"
+import { Notification } from "../realtime.d"
+import { setNotification } from "../state"
+import { title } from "./notifications"
+
+const InactiveNotificationModal = ({
+  notification,
+  setInactiveNotification,
+}: {
+  notification: Notification
+  setInactiveNotification: (notification: Notification | null) => void
+}) => {
+  const [, dispatch] = useContext(StateDispatchContext)
+  const { removeNotification } = useContext(NotificationsContext)
+
+  const closeModal = () => {
+    setInactiveNotification(null)
```

Comment:
note to self: this is gross, try to figure out a way to get rid of the redundancy

Reply:
Right now if `useVehicle....()` fails, it sets the result elsewhere, which is why this information is in two places. What if we could detect failure from the same place we detect success, and roughly do:
```
case useVehicle(selectedNotification) do
  undefined -> <Loading/>
  null -> <FailureModal/>
  vehicle -> <VPP/>
end
```
and we could remove the `inactiveNotificationContext` entirely.